docs/orcfax-feeds/feed-overview.md

```diff
@@ -53,77 +35,144 @@ For a feed of type CER, the naming convention is
+the duration of that agreement.
+
+While the integrator has committed to supporting the sponsored feed, and thereby
+facilitating its publication, projects which are not participating in the feed's
+sponsorship and yet utilize the on-chain data should do so at their own risk;
+feed sponsors may change or remove feed support at their own discretion and are
+not required to communicate these intentions to external parties.
+
+Orcfax has the following sponsored feeds:
+
+| Feed name | Start Date |
+| --------- | ---------- |
+| TBA       | TBA        |
+
+## Sunsetting sponsored feeds
+
+Projects which fail to adhere to subscription agreements risk disruptions to
+their feeds.
+
+Integrators will be given a two week window to come current on balances owing,
+afterwhich unpaid feeds will have publication suspended.
+
+## Showcase feeds
+
+In order to demonstrate the kinds of feeds which integrators can utilize in
+their dApps, Orcfax will periodically select feeds from its catalogue and
+highlight them through publication as showcase feeds. Orcfax sets no minimum or
+maximum number of showcase feeds that it may support.
+
+Orcfax is currently offering the following CEX showcase feeds:
+
+| Feed name | Start Date                 | End date |
+| --------- | -------------------------- | -------- |
+| ADA-BTC   | Sep 12th 2024 17:01:24 UTC | TBA      |
+| BTC-USD   | Sep 12th 2024 17:01:24 UTC | TBA      |
+
+Orcfax is currently offering the following DEX showcase feeds:
+
+| Feed name | Start Date                 | End date |
+| --------- | -------------------------- | -------- |
+| ADA-DJED  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| ADA-iUSD  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| ADA-USDM  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| AGIX-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| BOOK-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| BTN-ADA   | Sep 12th 2024 17:01:24 UTC | TBA      |
+| CERRA-ADA | Sep 12th 2024 17:01:24 UTC | TBA      |
+| COPI-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| HOSKY-ADA | Sep 12th 2024 17:01:24 UTC | TBA      |
+| HUNT-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| INDY-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| LENFI-ADA | Sep 12th 2024 17:01:24 UTC | TBA      |
+| LQ-ADA    | Sep 12th 2024 17:01:24 UTC | TBA      |
+| MIN-ADA   | Sep 12th 2024 17:01:24 UTC | TBA      |
+| SHEN-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| SNEK-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| WMTx-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+
+:::warning
+
+Unless otherwise specified, Orcfax makes no guarantees as to how long showcase
+feeds will be offered in this capacity; Orcfax may change or remove support for
+showcase feeds at its discretion and may do so without notice. Integrators who
+would like to take advantage of these feeds or are interested in learning more
+about our services should contact us via [Discord][discord] or <info@orcfax.io>.
```

Comment:
This email link wont build locally. It can be:

```
[info@orcfax.io](mailto:info@orcfax.ioss).
```

Or needs to use the reference style formerly on this page:

```
[email]: mailto:info@orcfax.io
```